views/treso/bulletinVersement/viewBulletinVersement.hbs

```diff
@@ -0,0 +1,176 @@
+<body class="hold-transition skin-green fixed sidebar-mini">
```

Comment:
J'ai vu cette page sur le PC de Pablo - mais poour les autres MR c'est cool d'ajouter un petit screen de cette page - ça facilite beaucoup la relecture

Reply:
C'est noté ! C'est vrai que ça sera un peu plus visuel haha